nautobot/extras/plugins/__init__.py

```diff
@@ -103,6 +104,9 @@ def ready(self):
         if template_extensions is not None:
             register_template_extensions(template_extensions)
 
+        # Register custom jinja filters
+        import_object(f"{self.__module__}.{self.jinja_filters}")
+
```

Comment:
This is still needed as the filters need to be imported in order for the django_jinja `@library` decorators contained within `jinja_filters.jinja_filters` to be registered.

Reply:
I'm not convinced on this yet, sorry - this would be `import_object("dummy_plugin.jinja_filters.jinja_filters")` which is a dynamic equivalent to `from dummy_plugin.jinja_filters import jinja_filters`, but there's no `jinja_filters` list in `dummy_plugin.jinja_filters` to import, so this isn't actually doing anything in the dummy_plugin case, yet it seems like those filters are still being discovered anyway?